src/components/MessageModal.jsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { useSelector, useDispatch } from "react-redux";
+import { Modal, Paper, Typography } from "@material-ui/core";
+
+const MessageModal = () => {
+  const { messageOpen, messageType } = useSelector((state) => state);
+  const dispatch = useDispatch();
+
+  return (
+    <Modal
+      open={messageOpen}
+      onClose={() => dispatch({ type: "CLOSE_MESSAGE" })}
+      aria-labelledby="modal-title"
+      aria-describedby="modal-description"
+    >
+      <Paper style={styles.modal}>
+        <Typography id="modal-title" variant="h4" style={{textTransform: "uppercase", marginBottom: "10px"}}>{messageType}</Typography>
+        <Typography id="modal-description" data-cy="message" variant="h6">Thank you for your inquiry, we'll be in touch soon!</Typography>
+      </Paper>
+    </Modal>
+  );
+};
+
+export default MessageModal;
+
+const styles = {
+  modal: {    
+    position: "absolute",
+    top: "50%",
+    left: "50%",
+    transform: "translate(-50%, -50%)",
+    width: "90%",
+    maxWidth: "400px",
+    padding: "20px",
+    border: "1px solid #000",  
+    borderRadius: "10px",
+    backgroundColor: "limeGreen",
```

Comment:
Not sure about the lime green color. Can we use a color in the palette we have for this project?

Reply:
Well the palette only has orange colors. I think about such messages more in terms of psychology rather than looks. Green = success, Red = failure. We will understand that all is good even if we don't read text. I insist on this one.